### DISCUSSION TITLE
Smartthings Devices stop working after a few days

### DISCUSSION THREAD

**Discussion by digitaldeli** [iklein99/homebridge-smartthings#164] [Q&A] [answered]
I have consistently had this problem. When I restart Homebridge everything works great. However after a few days Smarthings devices will stop working. They work fine if I use the Smartthings app, but will stop responding to Siri, Home app, or from the Homebridge web UI. 

The lights are EcoSmart bulbs. The other devices are Smartthings multi sensors and water sensors. Not too much in ST, but enough I'd like to find out why this is so flakey for me. 

Homebridge v1.6
Homebridge-Smartthings v1.5.8
Raspberry Pi for Homebridge
Homekit original system (have not upgraded to the new system that was just released...again)

**Comment by iklein99** [comment, marked as answer]
I don't think there is anything I can do about this.  I do not experience this and I don't have anyone indicating such an issue.  I'd look at the device you're running homebridge on as well as whatever you are using as your home kit hub.

**Comment by digitaldeli** [comment]
@iklein99 ,  that is what I expected. I don't expect you to fix it and suspected it was an edge case. It hasn't been a big problem and I just work around it when it has failed. I'm hoping Matter and the new Homekit will improve these issues. I may just replace affected bulbs with Hue bulbs, those are much more reliable and customizable anyway....when I can find a good deal.

**Reply by JeffGoldner** [reply]
I find I have to restart HB about once a week in any case. But right now, nothing ST is working.

**Comment by StackPilotDev** [comment]
Same thing happens to me. Difference is the devices continue to work through the Homebridge accessories tab but on the Apple Home app they all say No Response. Is there anything I can do in this scenario? So far I tried restarting the child bridge which is how i set up the smartthings plugin and restarted homebridge entriely but I still was not able to gett the devices to work through Apple Home.

Thoughts? I've curently uninstalled the plugin and reinstalling now to start fresh.

**Reply by coryabooth** [reply]
Have you cleared the HomeBridge caches.  Accessories and plugins?
